app/src/components/layout/navLinks/NavLinks.tsx

```diff
@@ -22,7 +22,7 @@ const NavLinks = () => {
             <Header.NavItem
                 role="link"
                 className="clickable"
-                data-cy="logout-btn"
+                data-testid="logout-btn"
```

Comment:
All good as it is but don't feel the need to add test to the attribute, if it's an ID for a component go for it. :)

Reply:
Ahh I can see what you've done above, but Id should be enough.

Reply:
I generally tend to prefer using `data-*` custom attributes where possible, as this is considered a best practice in cypress and other UI test frameworks - https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
Personally I think data-testid is the ideal attribute if we are going this route, as jest also has built in query selectors that use data-testid